crates/interledger-packet/src/packet.rs

```diff
@@ -783,7 +762,7 @@ mod test_packet_type {
     #[test]
     fn try_from_empty() {
         assert_eq!(
-            "Invalid Packet: Unknown packet type: None",
+            "PacketType data not found",
```

Comment:
```suggestion
            "PacketType not found",
```
String-nit: I don't think the word `data` is meaningful here; when parsing we cannot hope to find anything but data.